src/main/java/school/lemon/changerequest/java/fractionnumber/FractionNumber.java

```diff
@@ -0,0 +1,110 @@
+package school.lemon.changerequest.java.fractionnumber;
+
+public class FractionNumber {
+    private int dividend;
+    private int divisor;
+
+    public static FractionNumber ONE = new FractionNumber(1, 1);
+    public static FractionNumber ZERO = new FractionNumber(0, 1);
+
+    private void normalize() {
```

Comment:
There is no need to define functions in the order you use them — common practice is put class fields/functions/constructors in a following order:
1. `private/protected/default/public static final` fields
1. `private/protected/default/public static` fields
1. `private/protected/default/public final` fields
1. `private/protected/default/public` fields
1. constructors
1. methods